### DISCUSSION TITLE
External Map Theme

### DISCUSSION THREAD

**Discussion by troNpo** [devemux86/cruiser#436] [Q&A] [answered]
Hello, is it possible to add external map themes to the ones that the app offers for download? 
Thank you

**Comment by devemux86** [comment, marked as answer]
@troNpo

Cruiser supports external map themes:
- Enable the "Settings | Application | Advanced | Extended menu"
- Then the map menu shows the entry "External map theme" to open external themes (as zip)

**Comment by troNpo** [comment]
Sorry for my clumsiness, but after enabling the advanced settings, I can't find in the map menu, "external map theme" maybe it's for the pro version? 
<img src="https://github.com/devemux86/cruiser/assets/97335264/ba0ec34b-f38f-45de-a8d6-c008d378c09a" width="50%"/>

**Reply by devemux86** [reply]
@troNpo 

You need to enable the "Settings | Application | Advanced | **Extended menu**".

**Comment by troNpo** [comment]
I did it
<img src="https://github.com/devemux86/cruiser/assets/97335264/c7de920b-08cd-4dba-bcc2-66303429f40c" width="50%"/>

**Reply by devemux86** [reply]
@troNpo 

Please see carefully the option location: "Settings | Application | Advanced | Extended menu".
Also see the images:

<img src="https://github.com/devemux86/cruiser/assets/3484020/93c684bc-f1df-4dca-b090-af13fda39ecd" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/3fbbe31c-461b-44c6-8fe9-4eea3aff25a5" width="50%"/>

**Reply by troNpo** [reply]
Seen!! Thank you for your patience
